huaweicloud/services/vpn/resource_huaweicloud_vpn_gateway.go

```diff
@@ -87,15 +90,45 @@ func ResourceGateway() *schema.Resource {
 				Type:     schema.TypeList,
 				MaxItems: 1,
 				Elem:     GatewayEipSchema(),
-				Required: true,
+				Optional: true,
 				ForceNew: true,
+				Computed: true,
 			},
 			"slave_eip": {
 				Type:     schema.TypeList,
 				MaxItems: 1,
 				Elem:     GatewayEipSchema(),
-				Required: true,
+				Optional: true,
 				ForceNew: true,
+				Computed: true,
+			},
+			"access_vpc_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `The access VPC ID of the VPN gateway.`,
+			},
+			"access_subnet_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `The access subnet ID of the VPN gateway.`,
+			},
+			"er_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `The enterprise router ID to attach with to VPN gateway..`,
```

Comment:
ending with double `.`
```suggestion
				Description: `The enterprise router ID to attach with to VPN gateway.`,
```

Reply:
fixed